frontend/src/__tests__/cypress/cypress/support/e2e.ts

```diff
@@ -117,6 +117,14 @@ Cypress.on('uncaught:exception', (err) => {
   return true;
 });
 
+// Ignore 'Unexpected token <' errors from webpack-dev-server fallback in E2E tests
```

Comment:
This fix is already included in my PR: https://github.com/opendatahub-io/odh-dashboard/pull/4511